projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py

```diff
@@ -51,8 +51,18 @@ class Installer:
     git_server_admin_email = "vdkuser@vmware.com"
     git_server_repository_name = "vdk-git-repo"
 
-    def __init__(self):
+    def __init__(self, server_id: str = ""):
+        self.git_server_image = "gogs/gogs:0.12"
```

Comment:
Is it a good idea to hardcode this?

Reply:
It was hard-coded on line 247. I simply moved it here. My thought was to make it configurable. But that would be in separate change.